.factory/hosting.py

```
@@ -870,6 +883,12 @@ def pr_set_labels(self, p, add=(), remove=(), repo=None):
                           f"{self._base()}/changeRequests/{p}/comments/{m['id']}",
                           body={"resolved": True})
         for name in add:
+            # 幂等（对齐 remove 分支）：已有同名未 resolved 标记则跳过
+            # ——重试/双写场景重复 POST 会堆未 resolved 重复标记（php#17）
+            if hits := self._label_markers(p, name):
+                print(f"[hosting] add {name}: 已有未 resolved 标记（幂等跳过）",
```

Ruff (0.16.3)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)